package.json

```diff
@@ -26,26 +26,33 @@
   },
   "scripts": {
     "test": "mocha",
-    "lint": "gulp sasslint",
-    "docs": "gulp sassdoc",
-    "commit": "yarn test && yarn lint && yarn docs"
+    "lint:sass": "yarn lint:nofix --fix",
+    "lint:nofix": "stylelint '**/*.scss'",
+    "lint:other": "prettier --write '**/*.{json,md,yml}'",
+    "lint": "run-p lint:sass lint:other",
+    "docs": "sassdoc 'sass/**/!(layout|scale)/*.scss'",
+    "commit": "run-s lint test docs",
+    "watch:sass": "chokidar '**/*.scss' '**/.stylelintrc.yml' -c 'yarn lint:nofix'",
+    "watch:tests": "chokidar 'test/**' -c 'yarn test'",
+    "watch:docs": "chokidar 'sass/**/*.scss' './.sassdocrc' -c 'yarn docs'",
+    "watch": "run-p watch:sass watch:tests watch:docs"
```

Comment:
I'm not actually sure what watcher commands are useful in practice. Generally I prefer more control as a dev, setting my editor to lint on save, and running tests/sassdoc manually when desired. @mirisuzanne What do you think?

Reply:
That's how I like it too, though I'm not sure how to set my editor to lint with the project rules.

It was @stacyk that requested a watcher, and might have more thoughts on it.

Reply:
I'm surprised to hear that I am the only one (of three of us so far) who likes a watcher. I don't know what else the question is here? If the watcher is optional, then wouldn't it be nice to keep it (if it is working)? 

I'm running to lunch, but will pull this branch and test it when I return.

Reply:
@stacyk Yeah I'm fine having a watcher -- I just don't know what watched files or tasks you would find useful. Right now it's linting all Sass on any change to a Sass file (but not auto-fixing; just showing errors/warnings), running Sass tests on changes to test files (should that run on any Sass file change?), and generating Herman docs on any change to Sass source files.

Reply:
> running Sass tests on changes to test files (should that run on any Sass file change?)

@jgerigmeyer I guess while working on this project specifically I wanted tests to run upon save on an scss file that is used in any test file. I realize that is complicated and maybe not how it works. For instance, if I made a change to `sass/layout/query-utils` the tests in the text/layout/query-utils would also run so I can see if the change I made in the main file made any difference to a test's results. It is a small enough project to run all sass tests if it is too tricky to figure out which files are `@use`d in a test. Does that make sense?

Reply:
Sure, that makes sense. I'll set it to run all tests on any change to any Sass file.

Reply:
> 

my two cents: I kept thinking of having a watcher while working on tests. Sort of how we do it for JS testing in Metecho. other than that, linting manually works fine for me.